tests/source/issue-CastComments.rs

```diff
@@ -0,0 +1,98 @@
+/*****
```

Comment:
admittedly a nit, but please use all lowercase for the file names so that this file will be consistent with the others in this repo and general Rust conventions